plugins/resources/unixfilesystem/CMakeLists.txt

```diff
@@ -15,6 +15,10 @@ target_link_libraries(
 target_include_directories(
   irods_resource_plugin_obj-unixfilesystem
   PUBLIC
+  "${IRODS_EXTERNALS_FULLPATH_ARCHIVE}/include"
+  "${IRODS_EXTERNALS_FULLPATH_BOOST}/include"
+  "${IRODS_EXTERNALS_FULLPATH_FMT}/include"
+  "${IRODS_EXTERNALS_FULLPATH_SPDLOG}/include"
```

Comment:
I do not think these lines are needed for the object library target.

Reply:
The archive was not but the others seem to be required.